### PR TITLE
3.4 changelog

### PR DESCRIPTION
**Pull request by Bob-Murphy**
### What have you changed and why?
Contains 3.4 changelog for update release

### CODE REVIEW

File: changelog.rst [hunk 1]
```diff
@@ -3,6 +3,111 @@ Changelog
...
+.. admonition:: Minor
+
+   * :issue:`3012` & :issue:`3011` & :issue:`2978` Updated Antistasi based on ACE 3.16.0 / 3.16.1 updates
+   * :issue:`2997` Fixed SPE Panterfaust and mortyAI issues
+   * :issue:`2956` Moved Headless Clients with HQ to theoretically improve AI responses
+   * :issue:`2930` Adds Global Mobilization radiotowers to be compatible with Antistasi
+   * :issue:`2899` Enabled rebel AI to utilize rifle+muzzle combo grenade launchers
+   * :issue:`2876` PATCOM Garrison adjustments and minor fixes
+   * :issue:`2860` Improved Garage source checking
+   * :issue:`2842` Moved ACE Init
```
Comment (jaj22) [inline changelog.rst:51]: This one did also add some ACE event handling, notably the grenade throwing and injection.

File: changelog.rst [hunk 2]
```diff
@@ -3,6 +3,111 @@ Changelog
...
+   * :issue:`3027` Updated Arma version checking
+   * :issue:`3026` Fixed logistics JIP bugs
+   * :issue:`3021` Improved creteSupport logging
+   * :issue:`3016` Fixed broken UI config dependencies
+   * :issue:`3013` Fixed basic/medical gear missing in SPE/IFA templates
+   * :issue:`3008` & :issue:`3004` Fixed config errors for Vanilla/Mod assets
+   * :issue:`3001` Map Fixes - Fixed vehicle markers on Livonia, removed seaport_6 on Malden and moved outposts _1 and _14 on Malden
+   * :issue:`2998` Fixed High Command being visible after leaving High Command mode
+   * :issue:`2994` Fixed Spam in Zeus Logging
+   * :issue:`2987` Fixed theBoss war not being published in Autostart case
```
Comment (jaj22) [inline changelog.rst:88]: war -> var

File: changelog.rst [hunk 3]
```diff
@@ -3,6 +3,111 @@ Changelog
...
+   * :issue:`2979` Fixed non-basic backpack in SPE rebel template
+   * :issue:`2975` Fixed High Command Squad Icons being visible in 3D Display
```
Comment (jaj22) [inline changelog.rst:90]: Minor feature rather than bugfix.

File: changelog.rst [hunk 4]
```diff
@@ -3,6 +3,111 @@ Changelog
...
+   * :issue:`2885` Updated ChDKZ templates with retextured tanks
+
+.. admonition:: Localization
+
+   * [Multiple PRs] - Multiple hundreds of hard coded strings converted into strings
+   * [Multiple PRs] - Multiple thousand new translations accumulative for all the supported languages
+
+.. admonition:: Bugfixes
+
+   * :issue:`3027` Updated Arma version checking
```
Comment (jaj22) [inline changelog.rst:79]: Minor feature not bugfix.

File: changelog.rst [hunk 5]
```diff
@@ -3,6 +3,111 @@ Changelog
...
+   * :issue:`3026` Fixed logistics JIP bugs
+   * :issue:`3021` Improved creteSupport logging
+   * :issue:`3016` Fixed broken UI config dependencies
+   * :issue:`3013` Fixed basic/medical gear missing in SPE/IFA templates
+   * :issue:`3008` & :issue:`3004` Fixed config errors for Vanilla/Mod assets
+   * :issue:`3001` Map Fixes - Fixed vehicle markers on Livonia, removed seaport_6 on Malden and moved outposts _1 and _14 on Malden
+   * :issue:`2998` Fixed High Command being visible after leaving High Command mode
```
Comment (jaj22) [inline changelog.rst:86]: Bugfix for new stuff, don't include here.

File: changelog.rst [hunk 6]
```diff
@@ -3,6 +3,111 @@ Changelog
...
+.. admonition:: Localization
+
+   * [Multiple PRs] - Multiple hundreds of hard coded strings converted into strings
+   * [Multiple PRs] - Multiple thousand new translations accumulative for all the supported languages
+
+.. admonition:: Bugfixes
+
+   * :issue:`3027` Updated Arma version checking
+   * :issue:`3026` Fixed logistics JIP bugs
+   * :issue:`3021` Improved creteSupport logging
```
Comment (jaj22) [inline changelog.rst:81]: createSupport

File: changelog.rst (hunk 1 above)
Comment (Bob-Murphy) [reply changelog.rst]: fixed

File: changelog.rst (hunk 3 above)
Comment (Bob-Murphy) [reply changelog.rst]: fixed

File: changelog.rst (hunk 2 above)
Comment (Bob-Murphy) [reply changelog.rst]: fixed

File: changelog.rst (hunk 4 above)
Comment (Bob-Murphy) [reply changelog.rst]: fixed

File: changelog.rst (hunk 5 above)
Comment (Bob-Murphy) [reply changelog.rst]: fixed

File: changelog.rst (hunk 6 above)
Comment (Bob-Murphy) [reply changelog.rst]: fixed

File: changelog.rst [hunk 7]
```diff
@@ -3,6 +3,110 @@ Changelog
...
+   * :issue:`2686` Added ACE ropes to starting Arsenal
+
+.. admonition:: Template updates
+
+   * :issue:`2933` Militia offroads and Civilian CH-49 config entries and additions for Vanilla templates
+   * :issue:`2811` Added new 3CB AAF Desert/Brown template
+   * :issue:`2960` Added vehiclesLightTanks template category
+   * :issue:`2955` Updated RHS ChDKZ AI template with adjusted vehicle lineup and some gear changes
+   * :issue:`2953` Updated 3CB FIA and TKM rebel templates with starting Enfields
+   * :issue:`2951` Updated 3CB CNM rebel template with starting Mosin and Shtgun
```
Comment (ante185) [inline changelog.rst:65]: shotgun

Comment (Bob-Murphy) [reply changelog.rst]: fixed

File: changelog.rst [hunk 8]
```diff
@@ -3,6 +3,110 @@ Changelog
...
+
+.. admonition:: Bugfixes
+
+   * :issue:`3026` Fixed logistics JIP bugs
+   * :issue:`3021` Improved createSupport logging
+   * :issue:`3016` Fixed broken UI config dependencies
+   * :issue:`3013` Fixed basic/medical gear missing in SPE/IFA templates
+   * :issue:`3008` & :issue:`3004` Fixed config errors for Vanilla/Mod assets
+   * :issue:`3001` Map Fixes - Fixed vehicle markers on Livonia, removed seaport_6 on Malden and moved outposts _1 and _14 on Malden
+   * :issue:`2994` Fixed Spam in Zeus Logging
```
Comment (rautamiekka) [inline changelog.rst:87]:
```suggestion
   * :issue:`2994` Fixed spam in Zeus Logging
```

File: changelog.rst [hunk 9]
```diff
@@ -3,6 +3,110 @@ Changelog
...
+   * :issue:`2987` Fixed theBoss var not being published in Autostart case
+   * :issue:`2979` Fixed non-basic backpack in SPE rebel template
+   * :issue:`2959` Fixed RPT error when admin disconnects
+   * :issue:`2947` Fixed Huron not using gear on combat landing
+   * :issue:`2945` Added missing Land_Cargo_HQ_V4_F building to array of static spawners
+   * :issue:`2942` Autumn Cleaning - removed dead IFA templated, fixed a typo and small map issues on Anizay and Malden
```
Comment (rautamiekka) [inline changelog.rst:93]:
```suggestion
   * :issue:`2942` Autumn Cleaning - removed dead IFA templates, fixed a typo and small map issues on Anizay and Malden
```

File: changelog.rst [hunk 10]
```diff
@@ -3,6 +3,110 @@ Changelog
...
+   * :issue:`2922` Multiple Bug Fixes found by LordGolias using his sqf-analyzer
+   * :issue:`2917` Fixed AI not being able to use light helis in QRFs
+    * [Multiple PRs] small typos, in-production fixes for new systems and such
```
Comment (rautamiekka) [inline changelog.rst:96]:
```suggestion
   * [Multiple PRs] small typos, in-production fixes for new systems and such
```

**Review by rautamiekka — CHANGES_REQUESTED** [review]

File: changelog.rst (hunk 8 above)
Comment (Bob-Murphy) [reply changelog.rst]: fixed

File: changelog.rst (hunk 9 above)
Comment (Bob-Murphy) [reply changelog.rst]: fixed

File: changelog.rst (hunk 10 above)
Comment (Bob-Murphy) [reply changelog.rst]: fixed

**Review by jaj22 — APPROVED** [review]

**Pull request merged**